### DISCUSSION TITLE
Is sharing (embedded) disables on the self-hosted version?

### DISCUSSION THREAD

**Discussion by zeigerpuppy** [heyform/heyform#79] [Self hosting]
I have installed self-hosted and am attempting to test embedding a form in Hugo.
- as described in https://heyform.net/blog/2022/01/how-to-add-a-contact-form-to-hugo/

In that tutorial, it shows the "embed in a web page options":

![image](https://github.com/heyform/heyform/assets/415981/39db8f1b-a14c-40de-bb5f-71fb0aa11800)

However, on self-hosted, it doesn't look like this sharing option is available.  All that is shown in the share dialog is:

![Screenshot from 2024-05-05 12-28-17](https://github.com/heyform/heyform/assets/415981/a8757b55-5c63-49fd-bbe5-8ede5c739e2c)

Is this deliberately disabled, or is there some option that enables embedding?

**Comment by dearroy** [comment]
Hey @zeigerpuppy, apologies for the confusion. We recently updated the user interface but forgot to update the tutorial accordingly.

The share dialog should now be visible, but you'll need to upgrade to the latest release in order to access it.

**Reply by zeigerpuppy** [reply]
thanks, tried upgrade but I think there is an error in the upgraded build:

docker-compose log:

```
heyform    | TypeError: Class extends value undefined is not a constructor or null
heyform    |     at Object.<anonymous> (/app/src/utils/directives/lower.ts:4:41)
heyform    |     at Module._compile (node:internal/modules/cjs/loader:1364:14)
heyform    |     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
heyform    |     at Module.load (node:internal/modules/cjs/loader:1203:32)
heyform    |     at Module._load (node:internal/modules/cjs/loader:1019:12)
heyform    |     at Module.require (node:internal/modules/cjs/loader:1231:19)
heyform    |     at require (node:internal/modules/helpers:177:18)
heyform    |     at Object.<anonymous> (/app/src/utils/directives/index.ts:1:1)
heyform    |     at Module._compile (node:internal/modules/cjs/loader:1364:14)
heyform    |     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
heyform    |     at Module.load (node:internal/modules/cjs/loader:1203:32)
heyform    |     at Module._load (node:internal/modules/cjs/loader:1019:12)
heyform    |     at Module.require (node:internal/modules/cjs/loader:1231:19)
heyform    |     at require (node:internal/modules/helpers:177:18)
heyform    |     at Object.<anonymous> (/app/src/utils/index.ts:6:1)
heyform    |     at Module._compile (node:internal/modules/cjs/loader:1364:14)
heyform    | 
heyform    | Node.js v18.20.0
heyform exited with code 1
```

**Reply by iMuFeng** [reply]
Fixed in v0.0.10

**Comment by zeigerpuppy** [comment]
Ah great, thanks @dearroy !

I think there may have also been some other code changes with the embed options... looks like the `/embed` path now returns am html file rather than a script... Will do some more digging.
